### ISSUE TITLE
Error when creating virtual files and time series 

### ISSUE THREAD

**Issue by dk-forestry** [lbusett/MODIStsp#77]
I ran into an error for the products MCD12Q1 and MCD64A1. Downloading and processing seems to work fine, until this step: 

`Creating Virtual Files and rts time series for layer LC1`

This error occurs: 

`Error in grep(file_prefix, out_meta_files) : 
  object 'file_prefix' not found`

Interestingly, the processing step and the following error also occur when both `Create Virtual Rasters` and `Create RasterStacks` are set to `No`. 

I'm using version 1.3.2.9 by the way.

**Comment by lbusett** [comment]
Thanks for signaling this. A stupid bug affecting all "Combined" products (i.e., the `MCD` ones). 

Should be solved on commit https://github.com/lbusett/MODIStsp/commit/a0bb7e2a372df465153a87a6f329a45f49a2346b

By the way: what is Product "MCD64A1" ? I don't think we have it in our "list" of available products.

**Comment by dk-forestry** [comment]
Thanks, I'm now able to download and process MCD12Q1. 

MCD64A1 seems to be the collection 6-version of MCD45A1 (burned_monthly_500m). When I opened the issue, I must have gotten confused between the two products . :)

By the way, I noticed another bug for MCD45A1: no output files are generated for the layer `Burn_Date` (output folders are empty). Maybe this is related to [this issue](https://github.com/lbusett/MODIStsp/issues/60)?
